### PR TITLE
(Feat) 55-5 add speaker notes to unit tests presentation

### PR DESCRIPTION
**Pull request by PrykhodzkaV**

### CODE REVIEW

**Comment by Dzmitry-Klokau** [comment]
<section class="presentation-name-slide">
<h4>Unit tests</h4>
</section>

could you write in the asides main idea of this topic

**Comment by Dzmitry-Klokau** [comment]
```
<section class="agenda-slide">
          <h4 class="slide-name">AGENDA</h4>
```
please use this aside:
```
<aside class="notes">
            On this slide you can see today's agenda.
            <br /><br />
            We will start with..
            <br /><br />
            Next…
            <br /><br />
            And the last one ..
          </aside>
```

File: unit-testing/presentation.html [hunk 1]
```diff
@@ -197,18 +278,36 @@ <h4 class="slide-name">TDD VS BDD</h4>
               </tr>
             </tbody>
           </table>
+          <aside class="notes">
```
Comment (Dzmitry-Klokau) [inline unit-testing/presentation.html:281]: could you remove words BDD and TDD inside td-s?
example:
```
// <p>BDD is a team methodology</p>
// to
<p>team methodology</p>
```

to make slide lightly

**Comment by Dzmitry-Klokau** [comment]
```
section class="picture-slide">
             <h4 class="slide-name">Testing statistic​</h4>
```
could comment what we can understand from this picture

**Comment by Dzmitry-Klokau** [comment]
```
<section class="picture-slide">
            <h4 class="slide-name">Frameworks comparison</h4>
```

conclusions from the picture

**Comment by Dzmitry-Klokau** [comment]
```
<section class="bullets-slide">
          <h4 class="slide-name">Why Jest?</h4>
```
could you move slide text to the asides
and add Jest logo to the slide

**Comment by Dzmitry-Klokau** [comment]
```
<section class="text-and-conding-slide">
          <h4 class="slide-name">How to set up Jest?</h4>
```
could you move slide text to the asides
just left code on the slide

**Comment by Dzmitry-Klokau** [comment]
```
<section class="text-and-conding-slide">
           <h4 class="slide-name">Configuration file​</h4>
```
could you move slide text to the asides

**Comment by Dzmitry-Klokau** [comment]
```
<section class="text-and-conding-slide">
            <h4 class="slide-name">The Jest Object​</h4>
```
could you move slide text to the asides

**Comment by Dzmitry-Klokau** [comment]
```
 <section class="text-and-conding-slide">
          <h4 class="slide-name">Snapshots​​</h4>
```
could you move slide text to the asides

**Comment by Dzmitry-Klokau** [comment]
```
 <h4 class="slide-name">Test coverage​</h4>
```

could you left only key words in the slide text

**Comment by Dzmitry-Klokau** [comment]
```
<h4 class="slide-name">Covarage report​</h4>
```
```
<h4 class="slide-name">Covarage report in coverage/lcov-report/index.html​</h4>
```
```
<h4 class="slide-name">Statement not covered​</h4>
```
```
<h4 class="slide-name">When tests fail​​</h4>
```
could you comment what we can understand from this picture

**Comment by Dzmitry-Klokau** [comment]
```
<h4 class="slide-name">Husky</h4>
```
please move text to the asides

**Comment by Dzmitry-Klokau** [comment]
image
 really hard to read such text

could you add `<br/><br/>` between rows and `<strong>text</strong> to the main words?

**Review by annakosakova — APPROVED** [review]

**Review by Dzmitry-Klokau — APPROVED** [review]

File: unit-testing/presentation.html [hunk 2]
```diff
@@ -45,6 +45,12 @@
         <!-- 0 slide -->
         <section class="presentation-name-slide">
           <h4>Unit tests</h4>
+          <aside class="notes">
+            Main idea of this topic is get to know what testing is and what role it plays in software development.
+            <br /><br />
+            Also dive in unit testing, why developers have to write unit tests, have a look how to configure environment
```
Comment (unfalse) [inline unit-testing/presentation.html:51]: maybe "Also **to** dive in unit testing"

**Pull request merged**